lib/timetable-visualisation/PlanPreview.tsx

```diff
@@ -0,0 +1,74 @@
+import { DevelopmentPlan } from "../types/timetable";
+
+import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
+
+type PlanPreviewProps = {
+  plan: DevelopmentPlan;
+};
+
+export const PlanPreview = ({
+  plan: {
+    name,
+    description,
+    adoptedDate,
+    periodStartDate,
+    periodEndDate,
+    developmentPlanType,
+    developmentPlanGeography,
+    organisations,
+    entryDate,
+    startDate,
+    endDate,
+    documentationUrl,
+    timetableEvents,
+  },
+}: PlanPreviewProps) => {
+  return (
+    <div data-testid="plan-preview">
+      <div>
+        <h1 className="govuk-heading-xl" data-testid="form-title">
```

Comment:
this is the wrong testId for this element. I will remove

Reply:
tbh, I think we can add test IDs as and when we need them. A lot of the time we can find the element by text (e.g. finding a button by the text) rather than adding an ID.

Reply:
removed it 👍